corehq/apps/hqadmin/reporting/reports.py

```diff
@@ -805,14 +807,14 @@ def get_submitted_users():
     return real_form_users | real_sms_users
 
 
-def get_case_owner_filters():
+def get_case_owner_filters(domains):
     result = {'terms': {}}
 
-    mobile_user_ids = list(get_user_ids(True))
+    mobile_user_ids = list(get_user_ids(True, domains))
 
     def all_groups():
-        for domain in Domain.get_all():
-            for group in Group.by_domain(domain.name):
+        for domain in domains:
```

Comment:
I think this was getting every group that existed in hq everytime someone loaded admin reports...